diff --git a/hw/vendor/pulp_platform_gpio.core b/hw/vendor/pulp_platform_gpio.core
index 2d79c61..7485c2c 100644
--- a/hw/vendor/pulp_platform_gpio.core
+++ b/hw/vendor/pulp_platform_gpio.core
@@ -40,6 +40,7 @@ targets:
     - "gpio-test? (testbench)"
     - target_sim? (clock-gate)
     - target_sim_sc? (clock-gate)
+    - target_sim_rtl_tsmc65? (clock-gate)
     - target_asic_synthesis? (clock-gate)
     - target_asic_yosys_synthesis? (clock-gate)
     - target_nexys-a7-100t? (no-clock-gate)
